.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
this code usually do not commit into the repo, cause developers might have different IDE's for development
in our case we both use vscode, so it's ok